docs/index.rst

```diff
@@ -27,9 +27,21 @@ application orchestration capabilities to hosts at Edge.
    getting-started/roadmap.md
    getting-started/support.md
    getting-started/community-membership
-   getting-started/release_package
    getting-started/reporting_bugs
 
+.. toctree::
+   :maxdepth: 2
+   :caption: Setup
+
+   setup/requirements
+   setup/setup
```

Comment:
What is the relationship of the `setup/setup` and `setup/kubeedge_install` below? @anvithks